x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -198,7 +281,7 @@ export interface SafeResolverRelatedEvents {
  */
 export interface ResolverRelatedAlerts {
   entityID: string;
-  alerts: ResolverEvent[];
+  alerts: SafeResolverEvent[];
```

Comment:
The frontend isn't referencing the related alerts yet so switching it over to use the safe version.